packages/storage-ui/src/Components/Modules/FilesList/FilesList.tsx

```diff
@@ -779,15 +781,14 @@ const FilesList = () => {
                 </TableHeadCell>
                 <TableHeadCell
                   sortButtons={true}
-                  align="left"
                   onSortChange={() => handleSortToggle("name")}
                   sortDirection={column === "name" ? direction : undefined}
                   sortActive={column === "name"}
                 >
                   <Trans>Name</Trans>
                 </TableHeadCell>
                 {
-                  fileSystemType && fileSystemType !== "ipfs" && <TableHeadCell
+                  desktop && fileSystemType && fileSystemType !== "ipfs" && <TableHeadCell
```

Comment:
No need for desktop check here (thanks lgtm-bot)
```suggestion
                  fileSystemType && fileSystemType !== "ipfs" && <TableHeadCell
```